services/notification/tests/unit/test_notification_service.py

```diff
@@ -461,6 +462,7 @@ def test_notify_individual_notifier_timeout(self, mocker, sample_comparison):
     def test_notify_individual_checks_notifier(
         self, mocker, sample_comparison, mock_repo_provider, mock_configuration
     ):
+        mock_repo_provider.get_commit_statuses.return_value = Status([])
```

Comment:
instead of defining this in *every* test thats using the function, how about adding this as a default to the `mock_repo_provider`

Reply:
Not a bad thought, but the mock_repo_provider is used in other files/tests so it may lead to unnecessary calls/undesired side effects. I also like it this way cause it makes it clearer to me what each test does, I find it better for readability :). I'll merge this tmrw since it's kinda late now